### PR TITLE
Moved emagged var to obj/item parent.

### PR DESCRIPTION
**Pull request by ShizCalev**
Cleans up a bunch of copypastas

### CODE REVIEW

**Review by Cyberboss — APPROVED** [review]

**Comment by lzimann** [comment]
I am pretty sure this could even go to `/obj`

**Comment by kevinz000** [comment]
... memory issues..?

File: code/game/objects/items/cigs_lighters.dm [hunk 1]
```diff
@@ -630,7 +630,6 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	var/vapetime = 0 //this so it won't puff out clouds every tick
 	var/screw = 0 // kinky
 	var/super = 0 //for the fattest vapes dude.
-	var/emagged = FALSE //LET THE GRIEF BEGIN
```
Comment (Shadowlight213) [inline code/game/objects/items/cigs_lighters.dm]: Rip griff

**Comment by nicbn** [comment]
Less class members is less heap usage, don't.

**Comment by Cyberboss** [comment]
Duped code is worse

**Pull request merged**